Sources/UITextFieldDelegate/CurrencyUITextFieldDelegate.swift

```diff
@@ -14,17 +14,19 @@ import CurrencyFormatter
 
 /// Custom text field delegate
 public class CurrencyUITextFieldDelegate: NSObject {
-
+    
     public var formatter: CurrencyFormatterProtocol!
-
+    
     /// Text field clears its text when value value is equal to zero
     public var clearsWhenValueIsZero: Bool = false
-
+    
+    weak public var passthroughDelegate: UITextFieldDelegate?
```

Comment:
The passthrough delegate idea is totally an idea reminded to me by one of the other currency formatter projects [LionheartCurrencyTextField](https://github.com/lionheart/LionheartCurrencyTextField).  It's good to provide users a way to maintain all of their existing delegate code with minimal changes, while also gaining currency formatting capabilities.

I think it's good to maintaining things as UITextFieldDelegate connections:
-  Allows users to easily retain all existing delegate methods
- Allows users maximum flexibility to control the call order of a chain of delegates without having to account for any restricted protocols at any point.
- It's not likely that people would combine two text altering delegates, but maybe they can make it work out for themselves.

In our project we need most of the UITextFieldDelegate method calls.  Even the ones you would think could be dangerous due to possible text altering.  I view a limited protocol as putting unnecessary restrictions on how other people can easily use CurrencyText in a chain of delegates.  Years ago, I built some field validators that used chains of UITextFieldDelegates and it worked out really well.

Reply:
Ok, I do agree it can be useful if well used.
Therefore I checked out your branch and did just some minor adjustments to it:
- Documented this new property to improve clarity;
- Added an evaluation to prevent the user from setting the CurrencyUITextField delegate (self) as passthrough delegate.
- Removed the macOS checking (by now), because the package/framework is only available for iOS. It can be added in the future when supporting AppKit/macOS.

Thanks again for your nice contribution 🎉. I'm fixing the tests then this improvement can be merged and ready to be released in a patch.